templates/github/.github/workflows/update-labels.yml.j2

```diff
@@ -0,0 +1,38 @@
+{% include 'header.j2' %}
+{% from 'macros.j2' import
+  checkout,
+  setup_python,
```

Comment:
Is calling `setup_python` two times in this workflow required?

Reply:
It is imported and then called once AFAIK